graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -194,12 +418,17 @@ private String performDirectIndexQuery(String query, boolean source) throws Atla
         return EntityUtils.toString(response.getEntity());
     }
 
-    private DirectIndexQueryResult getResultFromResponse(String responseString) {
-        DirectIndexQueryResult result = new DirectIndexQueryResult();
-
+    private DirectIndexQueryResult getResultFromResponse(String responseString, boolean async) throws IOException {
```

Comment:
async param is not being used.

Reply:
It is intentional, it is for method overloading in Java